backend/src/cli.mjs

```diff
@@ -1,33 +1,48 @@
 import yargs from 'yargs';
-import {Request} from './request.mjs';
+import {defaultFormat, OPTION_FORMAT, OPTION_FROM_DATE, OPTION_TO_DATE, Options, supportedFormats} from './options.mjs';
 import {collectAllUsages} from "./collect-dynamodb-usage.mjs";
 import {renderCsv} from "./render/csv.mjs";
 import {renderHtml} from "./render/html.mjs";
 
 export default async () => {
-  // Parse user request
-  const request = new Request();
-  const options = yargs(process.argv.slice(2))
-    .option('format', {
-      type: 'string',
-      description: 'Output format',
-      choices: ['html', 'csv'],
-      default: 'html'
-    })
-    .version(false)
-    .parse();
-
-  if (options.format === 'html' || options.format === 'csv') {
-    request.withFormat(options.format);
-  }
+  // Parse program options
+  const options = parseOptions();
 
   // Collect DynamoDB usage
-  const usages = await collectAllUsages();
+  const usages = await collectAllUsages(options.dateRange);
 
   // Render result
-  if (request.format === 'csv') {
+  if (options.format === 'csv') {
     console.log(renderCsv(usages));
   } else {
     console.log(await renderHtml(usages));
   }
 };
+
+const parseOptions = () => {
+  let options;
+
+  yargs(process.argv.slice(2))
+    .option(OPTION_FORMAT, {
+      type: 'string',
+      description: 'Output format',
+      choices: supportedFormats,
+      default: defaultFormat
+    })
+    .option(OPTION_FROM_DATE, {
+      type: 'string',
+      description: `Start date to use when collecting usage of on-demand tables. Must be supplied if '--${OPTION_TO_DATE}' is supplied.`,
+    })
+    .option(OPTION_TO_DATE, {
+      type: 'string',
+      description: `End date to use when collecting usage of on-demand tables. Must be supplied if '--${OPTION_FROM_DATE}' is supplied.`,
```

Comment:
In the user facing doc you claim `By default, the end date is the date of today.`

Here you specify it is obligatory when `OPTION_FROM_DATE` is supplied.  Please clarify. Ideally it is fine to default to today if end date isn't gathered.